descriptors/ModuleDescriptor-template.json

```diff
@@ -26,9 +26,130 @@
           "pathPattern": "/_/tenant"
         }
       ]
+    },
+    {
+      "id": "patron-block-conditions",
+      "version": "0.1",
+      "handlers": [
```

Comment:
Shouldn't we change module's version when we add new interfaces?

Reply:
Since this module has not been released yet, I think we can leave the initial version